apps/rewards/app/store/events.js

```diff
@@ -29,7 +29,7 @@ export const handleEvent = async (state, event, settings) => {
       nextState = await onRefreshRewards(nextState, returnValues)
       break
     default:
-      console.log('[Rewards reducer] unhandled event:', event, returnValues)
+      // console.log('[Rewards reducer] unhandled event:', event, returnValues)
```

Comment:
Can we delete this comment, and with it the whole `default` section of the switch statement?

Reply:
Sure, but in this case it should be removed for all apps probably... ? Elsewhere we have, for example, this:
```
  default:
    break
```
Bad style?

Reply:
Given that I've previously advocated for removing seemingly-superfluous code from the script.js setup, and that code turned out to have a purpose after all, I suppose it's best to leave the `default` as-is. Also, this is a linting change, so it's probably best to keep a narrow focus and address only the linting issues.

Let's remove just this console line, then. No need to leave it present-but-commented.

Reply:
Sure!